.gitignore

```diff
@@ -5,4 +5,5 @@ node_modules
 *.tgz
 temp
 .idea
-yarn.lock
\ No newline at end of file
+yarn.lock
+stats.json
```

Comment:
Please dont ignore it

Reply:
Why not? The script has been in the repo for almost 2 years, and this file hasn't been committed. If there ends up being a use for it, it can be unignored

Reply:
I don't know why this was not added, But I am still in favor of adding it (whosoever has the CROWDIN_KEY)(perhaps GitHub actions)

even `electron` seems to have a similar stats.json - https://github.com/electron/i18n/blob/master/stats.json

Reply:
Looks like their version doesn't need the key https://github.com/electron/i18n/blob/master/script/stats.ts
I'm going to open a separate issue around wiring that up, but since this isn't functional today, and requires people to manually unstage the file, I'm going to leave the ignore

Reply:
electron/i18n doesn't need a Crowdin token because the Electron website acts as a proxy to the Crowdin API for unauthenticated requests. We did this to allow open-source contributors to access Crowdin metadata without needing a token.

See https://github.com/electron/electronjs.org/blob/master/CONTRIBUTING.md#crowdin-proxy

I think using a GitHub Action to fetch stats on a schedule would be ideal for this project.